datafusion/sqllogictest/test_files/joins.slt

```diff
@@ -4702,7 +4702,8 @@ physical_plan
 01)CrossJoinExec
 02)--DataSourceExec: partitions=1, partition_sizes=[0]
 03)--ProjectionExec: expr=[1 as Int64(1)]
-04)----PlaceholderRowExec
+04)----YieldStreamExec frequency=64
+05)------PlaceholderRowExec
```

Comment:
I have enabled the flag default to true, for all the slt/testing, only one case for adding YieldStreamExec, it's PlaceholderRowExec.

Do we also need to make PlaceholderRowExec as a built-in YieldStream? Or it's ok for us to add YieldStreamExec for this only cases?

cc @ozankabak @alamb